coalib/parsing/filters/CanDetectFilter.py

```diff
@@ -0,0 +1,21 @@
+from coalib.parsing.filters.FilterBase import FilterBase
+
+
+class CanDetectFilter(FilterBase):
+
+    def filter_section_bears(self, bears, args):
```

Comment:
don't have a problem to keep it like this currently, but do we need `self`? Then you don't need to instantiate a filter object^^